app/assets/javascripts/discourse/controllers/preferences/profile.js.es6

```diff
@@ -48,6 +50,31 @@ export default Controller.extend(PreferencesTabController, {
   },
 
   actions: {
+    showFeaturedTopicModal() {
+      showModal("feature-topic-on-profile", {
+        model: this.model,
+        title: "user.feature_topic_on_profile.title"
+      });
+    },
+
+    clearFeaturedTopicFromProfile() {
+      bootbox.confirm(
+        I18n.t("user.feature_topic_on_profile.clear.warning"),
+        result => {
+          if (result) {
+            return ajax(`/u/${this.model.username}/clear-featured-topic`, {
```

Comment:
Don't know much about bootbox usage, but what are the uses of these return here and in then ?

Reply:
Yeah, removing both returns does not change functionality.